packages/components/src/components/flyout-navigation/flyout-navigation.examples.md

```diff
@@ -0,0 +1,89 @@
+# Flyout Navigation
+
+<Notification heading="Experimental Component" state="warning">
+  Interface of Flyout Navigation might change in the near future.
+</Notification>
+
+<TableOfContents></TableOfContents>
+
+## Basic
+
+<Playground :markup="basicExample" :config="config"></Playground>
```

Comment:
Works in Storefront and in StackBlitz, but missing event handling (open/close).

Reply:
Implemented in the meantime.